### PR TITLE
"No results found" text fixed for dark mode

### LINKED ISSUE #30: "No result found" text is in black color in dark mode
### What is this About?
A bug that was generated due to #27 merged into main not taking into consideration the dark theme text colors. The "No results found" text is fine in light mode, but in dark mode, same colors were being used from the light mode, thus being difficult to read.

<br />

### Expected Behavior
Separate colors for light mode and dark mode.

<br />

### Current Behavior
Same color in both the modes

<br />

### Failure Information (for bugs)
The text color was hard-coded instead of being a variable(which can change dynamically depending on the current theme)

<br />

### Steps to Reproduce

<br />

### Context
The issue is due to absence of text color variable of the h2 tag used.

<br />

### Failure Logs
Untitled

This is the current condition in dark mode.

### PR DESCRIPTION
**Pull request by SubhamKarmakar24**
Fixes #30 

## Proposed Changes

  - The text is compatible for both dark and light modes

Current state
Untitled1

### CODE REVIEW

File: src/components/Results.css [hunk 1]
```diff
@@ -9,3 +9,8 @@
 .no-result {
   color: rgb(43, 43, 43);
 }
+
+.no-result-text {
+  color: var(--text-color-first);
+  transition: 0.4s;
```
Comment (faran4engg) [inline src/components/Results.css:15]: I think we dont need to create another class. `.no-result` css rule can be reused !

Comment (SubhamKarmakar24) [reply src/components/Results.css:15]: Yeah, I missed this trick. Fixing this in a moment @faran4engg

Comment (SubhamKarmakar24) [reply src/components/Results.css:11]: Done @faran4engg 
You can review the code

File: src/index.css [hunk 2]
```diff
@@ -27,7 +27,7 @@ input {
 
 .light {
   --background-color: #eaedf0;
-  --text-color-first: #000000;
+  --text-color-first: rgb(43, 43, 43);
```
Comment (faran4engg) [inline src/index.css:30]: @SubhamKarmakar24 , `rgb(43, 43, 43)` is something I added. In my opinion we can keep the legacy setting ( i.e. `#000000`).
Basically, we can keep changes in this file untouched.
Code changes in `src/components/Results.css` are good enough.

Comment (SubhamKarmakar24) [reply src/index.css:30]: Yes, fixed it through #32

**Pull request merged**